ceph/rados/core_workflows.py

```diff
@@ -410,19 +410,25 @@ def bench_write(self, pool_name: str, **kwargs) -> bool:
             3. max_objs -> max number of objects to be written (int)
             4. verify_stats -> arg to control whether obj stats need to
             be verified after write (bool) | default: True
+            5. rados_write_duration -> Duration for which the bench should be written on pool
+            6. check_ec -> If true, the method will wait to collect the exit status of the run, else will return
         Returns: True -> pass, False -> fail
         """
         duration = kwargs.get("rados_write_duration", 200)
         byte_size = kwargs.get("byte_size", 4096)
         max_objs = kwargs.get("max_objs")
         verify_stats = kwargs.get("verify_stats", True)
+        check_ec = kwargs.get("check_ec", True)
         cmd = f"sudo rados --no-log-to-stderr -b {byte_size} -p {pool_name} bench {duration} write --no-cleanup"
         org_objs = self.get_cephdf_stats(pool_name=pool_name)["stats"]["objects"]
         if max_objs:
             cmd = f"{cmd} --max-objects {max_objs}"
 
         try:
-            self.node.shell([cmd])
+            if not check_ec:
+                self.node.shell([cmd], check_status=False)
+            else:
+                self.node.shell([cmd])
```

Comment:
the if-else block is not needed, `self.node.shell([cmd], check_status=check_ec)` is enough

Reply:
Made changes.